bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -182,12 +182,12 @@ public String getScreenReaderText() {
         if (AssistPriority.LABEL.equals(assistPriority)) {
             Label label = getLabel();
             if (label != null) {
-                screenReaderText = "$label.$value";
+                screenReaderText = label.getValue();
```

Comment:
Since label can be dynamic, thats why this was added as an expression. Ideally, this needs to be evaluated by af-core dynamically at runtime.

Reply:
Can you open a backlog ticket to fix this ? For now, let's keep the value as static